### PR TITLE
Send a copy of repo manifest through garage-push

### PR DESCRIPTION
**Pull request by lbonn**
Depends on ~~https://github.com/advancedtelematic/aktualizr/pull/1438~~
https://github.com/advancedtelematic/aktualizr/pull/1440

### CODE REVIEW

**Comment by lbonn** [comment]
One concern I have right now is that this PR is coupled with the aktualizr change: as we're using a recently added cli flag, the garage-push command will fail if used on an older aktualizr version.

This is not a major issue but can be inconvenient nonetheless, I'll try to see if there is a clean workaround for that.

**Comment by patrickvacek** [comment]
> One concern I have right now is that this PR is coupled with the aktualizr change: as we're using a recently added cli flag, the garage-push command will fail if used on an older aktualizr version.

What's the problem as long as we bump the aktualizr version as part of this PR? I think it's fine.

**Comment by lbonn** [comment]
> What's the problem as long as we bump the aktualizr version as part of this PR? I think it's fine.

It's a bit more coupling than necessary and could be annoying if we're trying to bisect some issue on aktualizr for example. But yes, not major.

Still, I just pushed a small change to first check if the option shows up in `--help`. If you think it makes things worse, we can drop it.

File: classes/image_types_ostree.bbclass [hunk 1]
```diff
@@ -188,7 +188,8 @@ do_image_ostreepush[depends] += "aktualizr-native:do_populate_sysroot ca-certifi
 IMAGE_CMD_ostreepush () {
     # send a copy of the repo manifest to backend if available
     local SEND_MANIFEST=""
-    if [ -f ${IMAGE_ROOTFS}${sysconfdir}/manifest.xml ]; then
+    # check if garage-push supports the --repo-manifest option before trying
+    if $(garage-push --help | grep -q '^\s*--repo-manifest') && [ -f ${IMAGE_ROOTFS}${sysconfdir}/manifest.xml ]; then
```
Comment (patrickvacek) [inline classes/image_types_ostree.bbclass]: Ha, nice!

**Pull request merged**